docs/Reference/API-Methods.md

```diff
@@ -4484,6 +4484,43 @@ Returns full trace of all invoked opcodes of all transactions included in the bl
     methods, use the [`--rpc-http-api`](CLI/CLI-Syntax.md#rpc-http-api) or
     [`--rpc-ws-api`](CLI/CLI-Syntax.md#rpc-ws-api) options.
 
+### miner_changeTargetGasLimit
```

Comment:
What if no `--target-gas-limit` CLI was used before, what is the response of this method?

Reply:
When I tested this I received an error stating no target gas limit has been set.

Reply:
The error told me exactly what the issue was, so it was clear enough.